browser/resources/settings/brave_settings_overrides.js

```diff
@@ -0,0 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+window.BravePatching.registerPolymerBehaviors({
+  'settings-clear-browsing-data-dialog': [
```

Comment:
IIRC, this name has to match the component name exactly, right?

Reply:
Yes @bsclifton. If I know what you're thinking, I'm guessing it's to validate that that is a name of a recognised chromium Component name? Unfortunately this function neccessarily gets called before the component is registered, which could happen at any time. There isn't a definitive time where we could say "OK, all the components have been registered, let's check that the expected ones still exist". Although we could try a DOM timing even, such as DOMContentLoaded, or even a manual debug check that happens after 10 seconds......?!